client/blocks/reader-related-card-v2/index.jsx

```diff
@@ -95,6 +98,9 @@ export function RelatedPostCard( { post, site, onPostClick = noop, onSiteClick =
 		'has-thumbnail': !! featuredImage
 	} );
 
+	// TODO: is this okay?
+	const style = { WebkitLineClamp: lineClamp };
```

Comment:
Just finished doing some research because initially I didn't know what the problem with it being proprietary was.  Intuitively enough, WebkitLineClamp only works for webkit browsers (opera, safari, chrome).  

Since we use line-clamp in a few places, I rather add the mixin and solve the issue calypso-wide in a separate [issue](https://github.com/Automattic/wp-calypso/issues/9672)